.github/workflows/build-images-docs-builder.yaml

```diff
@@ -0,0 +1,143 @@
+name: Docs-builder Image Build
+
+# Any change in triggers needs to be reflected in the concurrency group.
+on:
+  pull_request:
```

Comment:
If we are ok with this, this will only work for PRs opened from a branch in this repository.